images_test.go

```diff
@@ -22,6 +22,12 @@ func TestRetaggedName(t *testing.T) {
 			},
 			expectedName: "quay.io/giantswarm/prometheus",
 		},
+		{
```

Comment:
This isn't needed for the retag, I just wanted to check that the retagger can correctly rename docker library images